integrations/python/pymongo/run-for-all.sh

```diff
@@ -0,0 +1,9 @@
+for file in ../../../tests/*
+do
+   if [[ $file == *.yml ]]
+   then
+      echo $file
+      python3 workload-executor.py "user:password@localhost" "$file"
```

Comment:
"user:password@localhost" is not a valid connection string because it's missing "mongodb://". How does this work?

Reply:
I am honestly not sure how it works. Could it mean that my routing in pymongo is incorrect?